news/2 Fixes/7993.md

```diff
@@ -0,0 +1 @@
+Let code in 'jupyter.runStartupCommands' work with intellisense.
```

Comment:
A big win for our power users.

Reply:
I'd put a few more comments here about what this means, so that its more obvious in the read me.
E.g. 
`import pandas` in startup command will import pandas & when you go into a cell and type `pandas.` then you immediately get code completions & the pandas is imported.

Would be a great way to re-advertise this feature for users.
I.e. now it works in runtime as well as intellisense.